src/scripts/main.js

```diff
@@ -1 +1,88 @@
 'use strict';
+
+let bodyDoc;
+let rightClick;
+let leftClick;
+
+if (document.querySelector('body') !== null) {
+  bodyDoc = document.querySelector('body');
```

Comment:
Querying the 'body' element twice is inefficient. You can assign the result of 'document.querySelector('body')' directly to 'bodyDoc' without the if-check since 'body' should always exist in a valid HTML document.